es-notebooks/build.gradle

```diff
@@ -224,18 +224,6 @@ integTest.getClusters().forEach{c -> c.plugin(project.getObjects().fileProperty(
 
 testClusters.integTest {
     testDistribution = 'OSS'
-    // need to install job-scheduler first, need to assemble job-scheduler first
-    plugin(provider(new Callable<RegularFile>(){
-        @Override
-        RegularFile call() throws Exception {
-            return new RegularFile() {
-                @Override
-                File getAsFile() {
-                    return fileTree("src/test/resources/job-scheduler").getSingleFile()
-                }
-            }
-        }
-    }))
```

Comment:
Any reason to remove this?

Reply:
es-notebooks setup was copied from reporting scheduler, which depends on job-scheduler. notebooks doesn't use it